src/app/view-modes/text-text/text-text.component.html

```diff
@@ -11,6 +12,7 @@
         <evt-text-panel 
             [pageID]="currentPageID$ | async" 
             [editionLevelID]="currentEditionLevels[1]"
+            [hidePageSelector]="currentEditionLevels[1] === 'critical'"
             (pageChange)="changePage($event)"     
             (editionLevelChange)="changeEditionLevel($event, 2)"
         ></evt-text-panel>
```

Comment:
Why in this case if it is critical edition level you don't use the `evt-critical-text-panel`?

Reply:
It's a mistake. In critical edition the text-text view mode is disabled. Do you agree or could it be useful?

Reply:
I don't know. You can leave it like as it is and we will change it in the future if we encounter such a problem.